### ISSUE TITLE
[OCULUS] Right Stick not scrolling text

### ISSUE THREAD

**Issue by Battou2501** [Nibre/MotherVR#185]
So using Oculus I cant scroll text on monitors, RS (Right Stick) is shown to do this, but is not working.

**Comment by Bluscream** [comment]
Maybe you need to press it down before scrolling?

**Comment by Battou2501** [comment]
> Maybe you need to press it down before scrolling?

No. Because using Oculus pressing the right stick is Back/Return, so I just exiting that text.

**Comment by Bluscream** [comment]
ahh i see, weirdly enough i have no problems using oculus quest (link) + oculus touch controllers

**Comment by JanSob** [comment]
Using a gamepad is much more intuitive in this game in my opinion. It doesn't have motion controls anyways.
